tasks.adoc

```diff
@@ -95,3 +95,13 @@ v rozumném čase (viz zadání) pracovat s většími mapami.
 
 Zadání a testy::
   https://github.com/cvut/beeclust/tree/speedup[github.com/cvut/beeclust @speedup]
+
+== Úloha filabel (testování): 3.12.2018
+
+Vaším úkolem za 5 bodů je napsat k vašemu dosavadnímu projektu *filabel* dokumentaci 
+pomocí http://www.sphinx-doc.org[Sphinx], která bude popisovat, jak s aplikací pracovat i
+jak funguje uvnitř, a to včetně doctestů, které poběží také na https://travis-ci.org[Travis CI]
+společně s testy z minulého úkolu.
+
+Zadání::
+  https://github.com/cvut/filabel/tree/tests[github.com/cvut/filabel @docs]
```

Comment:
```suggestion
  https://github.com/cvut/filabel/tree/docs[github.com/cvut/filabel @docs]
```

Reply:
Fixed, sry... all OK now? Can I merge & push to gitlab?